pkg/tenant/tenant_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/rotationalio/ensign/pkg/tenant/config"
 	"github.com/rotationalio/ensign/pkg/utils/emails"
 	"github.com/rotationalio/ensign/pkg/utils/logger"
+	emock "github.com/rotationalio/go-ensign/mock"
```

Comment:
Using the mock from the SDK library solves the mocking issue with the server mock.